docs/use-cases/ros_packages/configure_repository.rst

```diff
@@ -10,10 +10,10 @@ Setup Repository CI configuration (GitHub)
 The script **has to be executed** from the *main* folder of your package.
 
 .. code-block:: bash
-   :caption: Usage of script for setting up new packages.
+   :caption: Usage of the script for setting up new packages.
    :name: setup-package-ci
 
-   setup-repository-ci "repo_name" "repo_namespace" ["first_package" "second_package" ...]
+   setup-repository-ci "repo_name" "repo_namespace" ["first_package second_package ..."]
```

Comment:
I am not quite sure what's the best way to solve this: If user doesn't provide "first_package second_package ..." this is set to none which later on results in `$PKG_NAME$` not substituted inside some workflows (e.g. [CI-github_ci-ros-lint.yml](https://github.com/StoglRobotics/ros_team_workspace/pull/49/files#diff-2f6c1f68ed390ec52cdc7f2fc2e7b5b5d749ac7dd69a79bcb41aca9bb8fecf42)). This in return results in some workflow failing with a none too obvious error message, at least in my opinion. Either:
- Warn user about this?
- Make at least one package mandatory?
- Automatically detect packages and let user select?